tests/pr_test.go

```diff
@@ -38,81 +55,334 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func setupBasicExampleOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:            t,
-		TerraformDir:       dir,
-		Prefix:             prefix,
-		ResourceGroup:      resourceGroup,
-		BestRegionYAMLPath: "../common-dev-assets/common-go-assets/cloudinfo-region-scc-prefs.yaml",
-		TerraformVars: map[string]interface{}{
-			"access_tags": permanentResources["accessTags"],
+// Test the fully-configurable DA with defaults (no KMS encryption on bucket)
+func TestFullyConfigurable(t *testing.T) {
+	t.Parallel()
+
+	var region = validRegions[rand.Intn(len(validRegions))]
+
+	// ------------------------------------------------------------------------------------
+	// Provision COS, Sysdig, WP and EN first
+	// ------------------------------------------------------------------------------------
+
+	prefix := fmt.Sprintf("scc-da-%s", strings.ToLower(random.UniqueId()))
+	realTerraformDir := preReqDir
+	tempTerraformDir, _ := files.CopyTerraformFolderToTemp(realTerraformDir, fmt.Sprintf(prefix+"-%s", strings.ToLower(random.UniqueId())))
+	tags := common.GetTagsFromTravis()
+
+	// Verify ibmcloud_api_key variable is set
+	checkVariable := "TF_VAR_ibmcloud_api_key"
+	val, present := os.LookupEnv(checkVariable)
+	require.True(t, present, checkVariable+" environment variable not set")
+	require.NotEqual(t, "", val, checkVariable+" environment variable is empty")
```

Comment:
Seems like this code is not being used, so can be removed.

Reply:
@maheshwarishikha are you talking about line 77? This checks if the `TF_VAR_ibmcloud_api_key` env var value is set to an empty string, and fails if that is the case

Reply:
Line 73 - 77 is checking that the `TF_VAR_ibmcloud_api_key` env var is set as that is needed for the terraform apply to work

It has nothing to do with region selector